app/assets/scripts/actions/compare.js

```diff
@@ -72,6 +78,41 @@ export function fetchCompareLocationIfNeeded(index, location, filters = {}) {
   };
 }
 
+// Fetches random location and dispatches them to the compare slice of state.
+export function fetchRandomCompareLocs(limit = 2) {
+  return dispatch => {
+    // Dispatch all requests.
+    for (let index = 0; index < limit; index++) {
+      dispatch(requestCompareLocation(index));
+    }
+
+    const f = buildAPIQS({
+      order_by: 'random',
+      parameter: 'pm25',
```

Comment:
These names are currently being updated. You can pass the parameter id here instead and that will remain constant